Add 'expireAt' To Set TTL With Date Object

garrett-green opened the pull request:
Creates `expireAt` method on Repository, adding option to set TTL of entity/entities using a Date object (as requested in issue https://github.com/redis/redis-om-node/issues/120 ).

garrett-green commented:
@guyroyse Would it be possible for you to please take a look and weigh-in on this PR?

guyroyse commented on lib/repository/repository.ts:
```diff
@@ -259,6 +259,43 @@ export class Repository {
+    await Promise.all(
+      ids.map((id) => {
+        const key = this.makeKey(id);
+        return this.client.expire(key, ttlInSeconds);
```
There is a native `EXPIREAT` command in Redis. I think it would be more efficient and tidier to add a `.expireAt` method to `Client` and then call it.

That said, this implementation looks great!

garrett-green replied:
@guyroyse Thanks for reviewing and leaving feedback!

 I just updated the branch/PR to utilize the Redis' `EXPIREAT` via the client's `.expireAt` method and simplified the handling logic in the repository.

Please let me know if you think it need any other updates!

guyroyse reviewed (changes requested):
This looks really good. Just one bit of feedback and then I'll merge it an publish a new version. Sorry it's taken so long for me to respond but thanks so much for a solid contribution!

garrett-green commented:
@guyroyse Thank you for reviewing and leaving feedback!

 I just updated the branch/PR to utilize the Redis' `EXPIREAT` via the client's `.expireAt` method and simplified the handling logic in the repository.

Please let me know if you think it need any other updates!

guyroyse reviewed (approved):
LGTM

guyroyse commented:
Merged and published. https://www.npmjs.com/package/redis-om
